src/main/java/org/coffeetrain/doggifs/DogGifRepository.kt

```diff
@@ -0,0 +1,31 @@
+package org.coffeetrain.doggifs
+
+import android.content.Context
+import java.io.InputStream
+import java.util.Arrays
+
+class DogGifRepository(val context: Context) {
+  private val assetGifs = Arrays.asList("pounce", "weiners")
+  private var currentIndex = 0
+
+  fun getGif(handle: String): ByteArray {
+    val gifStream: InputStream = context.assets.open(handle + ".gif")
+    return gifStream.readBytes(5000000);
+  }
+
+  fun nextGifHandle(): String {
```

Comment:
this is kinda dangerous right? implicitly advancing the handle means you could accidentally skip.

maybe this should take a handle as a parameter?